src/components/team-stats/TeamGameStatsList.js

```diff
@@ -0,0 +1,18 @@
+import { Fragment } from "react";
+import classes from "./TeamGameStats.module.css";
+
+const TeamGameStatsList = (props) => {
+  const allTeamsStatsArray = props.gameStats;
+
+  const content = allTeamsStatsArray.map((stat) => (
+    <li key={stat.category} className={classes.list}>
+      <div>{stat.category}</div>
+      <div>{stat.homeTeamValue}</div>
+      <div>{stat.visitorTeamValue}</div>
+    </li>
+  ));
+
+  return <Fragment>{content}</Fragment>;
```

Comment:
use ul/ol tag if you're using li tag for each item

Reply:
the ul tag is in the parent component: TeamGamesStats.js.
Do you think it's confusing or bad practice?